portals/application/modules/userweb/controllers/AuthController.php

```diff
@@ -33,12 +34,25 @@ public function postAction()
         $result['ipRegistered'] = false;
 
         $terminal = $user->getTerminal();
+        $company = $user->getCompany();
 
         if (!empty($terminal)) {
 
             $dbAdapter = Zend_Db_Table::getDefaultAdapter();
             $terminalName = $terminal->getName();
 
+            $gsQRCodeEnabled = $user->getGsQRCode();
+            $hasFeatureQRCode = $company->hasFeature(Features::QRCODE);
+            $gsQRCode = ($gsQRCodeEnabled AND $hasFeatureQRCode) ? 1 : 0;
```

Comment:
In order to keep current code conventions, could you replace 'AND' by '&&' please?